main.go

```diff
@@ -174,11 +185,21 @@ func runRecordings(conf *Config, camera *lepton3.Lepton3) error {
 
 		// If motion detected, allow minFrames more frames.
 		if motion.Detect(frame) {
-			if window.Active() {
+			writeLog := motionLogFrame <= totalFrames-(10*framesHz)
+			if !window.Active() {
+				if writeLog {
+					motionLogFrame = totalFrames
+					log.Print("motion detected but outside of recording window")
+				}
+			} else if enoughSpace, err := diskSpaceCheck(conf.MinDiskSpace); !enoughSpace || err != nil {
+				if err != nil {
+					return err
+				} else if writeLog {
+					motionLogFrame = totalFrames
+					log.Print("motion detected but not enough space on disk to start recording")
```

Comment:
to make it more consistent with the earlier log message: "motion detected but not enough free disk space to start recording"